state-manager-frontend/src/app/page.tsx

```
@@ -0,0 +1,504 @@
+              currentStep={currentStep}
+              onStepClick={(stepIndex) => executeWorkflowStep(stepIndex)}
+            />
+          </div>
+        )}
+
+        {activeTab === 'nodes' && (
+          <div>
+            <h2 className="text-2xl font-bold text-gray-900 mb-6">Registered Nodes</h2>
+            <div className="grid gap-6">
+              {registeredNodes.map((node, index) => (
+                <div key={index} className="cursor-pointer" onClick={() => handleOpenNodeModal(node)}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)